xarray/core/dataset.py

```diff
@@ -1131,6 +1131,34 @@ def to_netcdf(self, path=None, mode='w', format=None, group=None,
                          engine=engine, encoding=encoding,
                          unlimited_dims=unlimited_dims)
 
+    def to_zarr(self, store=None, mode='a', synchronizer=None, group=None,
+                encoding=None):
+        """Write dataset contents to a zarr group.
+
+        Parameters
+        ----------
+        store : MutableMapping or str, optional
+            Store or path to directory in file system.
+        mode : {‘r’, ‘r+’, ‘a’, ‘w’, ‘w-‘}
+            Persistence mode: ‘r’ means read only (must exist); ‘r+’ means
+            read/write (must exist); ‘a’ means read/write (create if doesn’t
+            exist); ‘w’ means create (overwrite if exists); ‘w-‘ means create
+            (fail if exists).
+        synchronizer : object, optional
+            Array synchronizer
```

Comment:
This is a zarr concept?

Reply:
[Yes](http://zarr.readthedocs.io/en/master/tutorial.html#parallel-computing-and-synchronization). If xarray can guarantee that all writes will be aligned with chunk boundaries, then there is no need for a synchronizer.